src/appleseed/renderer/modeling/environmentedf/oslenvironmentedf.cpp

```diff
@@ -223,6 +516,25 @@ DictionaryArray OSLEnvironmentEDFFactory::get_input_metadata() const
                     .insert("shader_group", "Shader Groups"))
             .insert("use", "optional"));
 
+    metadata.push_back(
+        Dictionary()
+            .insert("name", "importance_map_size")
+            .insert("label", "Importance Map size")
+            .insert("type", "numeric")
+            .insert("min",
+                Dictionary()
+                    .insert("value", "1")
+                    .insert("type", "soft"))
+            .insert("max",
+                Dictionary()
+                    .insert("value", "65536")
+                    .insert("type", "soft"))
+            .insert("default", "1024")
+            .insert("use", "optional")
+            .insert("help", "Importance map size. width = size, height = size / 2"));
```

Comment:
Nice! I suggest tweaking it a bit (replaced period by colon, change spacing):

    "Importance map size: width = size, height = size/2"

Reply:
Agree, done.